superset/db_engine_specs/ocient.py

```diff
@@ -42,48 +53,66 @@ def _polygon_to_json(polygon: _STPolygon) -> str:
     return json_value
 
 def _linestring_to_json(linestring: _STLinestring) -> str:
+    """
+    Converts the _STLinestring object into its JSON representation.
+
+    :param data: the linestring object
+    :returns: JSON representation of the linestring
+    """
     return f'{str([[p.long, p.lat] for p in linestring.points])}'
 
 def _point_to_comma_delimited(point: _STPoint) -> str:
-    return f'{point.long}, {point.lat}'
-
+    """
+    Returns the x and y coordinates as a comma delimited string.
 
-# Map of datatypes that are not allowed in superset, but are allowed in our DB
-# Key is datatype, value is function to map, in order to convert to a valid superset type
+    :param data: the point object
+    :returns: the x and y coordinates as a comma delimited string
+    """
+    return f'{point.long}, {point.lat}'
 
-_superset_conversion_dict = {
-    _STPoint: _point_to_comma_delimited,
-    _STLinestring: _linestring_to_json,
-    _STPolygon: _polygon_to_json,
-    IPv4Address: str,
-    IPv6Address: str,
-    bytes: _to_hex
+# Sanitization function for column values
+SanitizeFunc = Callable[[Any], Any]
+
+# Represents a pair of a column index and the sanitization function 
+# to apply to its values.
+PlacedSanitizeFunc = NamedTuple(
+    "PlacedSanitizeFunc",
+    [
+        ("column_index", int),
+        ("sanitize_func", SanitizeFunc),
+    ],
+)
+
+# This map contains functions used to sanitize values for column types 
+# that cannot be processed natively by Superset.
+# 
+# Superset serializes temporal objects using a custom serializer 
+# defined in superset/utils/core.py (#json_int_dttm_ser(...)). Other 
+# are serialized by the default JSON encoder.
+_sanitized_ocient_type_codes: Dict[int, SanitizeFunc] = {
+    TypeCodes.BINARY: _to_hex,
+    TypeCodes.ST_POINT: _point_to_comma_delimited,
+    TypeCodes.IP: str,
+    TypeCodes.IPV4: str,
+    TypeCodes.ST_LINESTRING: _linestring_to_json,
+    TypeCodes.ST_POLYGON: _polygon_to_json,
```

Comment:
Any reason why these aren't in WKT format? i.e. just calling `str()`

Reply:
Superset does not support WKT for maps. These output geojson strings instead.